bitcoind/Dockerfile

```diff
@@ -2,21 +2,17 @@
 FROM debian:buster-slim as builder
 
 ARG BITCOIN_VERSION
-ARG BITCOIN_SHA256
-
-ENV BITCOIN_URL https://bitcoin.org/bin/bitcoin-core-${BITCOIN_VERSION}/bitcoin-${BITCOIN_VERSION}-x86_64-linux-gnu.tar.gz
+ARG SHA256_AMD64
```

Comment:
You could have only one arg BITCOIN_SHA256 and switch it in the build config, based on the arch?

Reply:
Done !

Reply:
How is it done?